src/integrations/Appcues/browser.js

```diff
@@ -11,6 +12,7 @@ class Appcues {
     this.analytics = analytics;
     this.accountId = config.accountId;
     this.apiKey = config.apiKey;
+    this.proxyUrl = config.proxyUrl;
```

Comment:
Let's call it `nativeSdkUrl` to be precise.

Reply:
updated